flask_securest/userstores/examples/README.md

```diff
@@ -0,0 +1 @@
+# This folder contains user store implementation examples, not official implementations.
```

Comment:
i'd remove the "#" cause it makes it all a giant title font, but this is a matter of taste